### ISSUE TITLE
Congenital human papillomavirus infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6854]
**Preferred term label:**
Congenital human papillomavirus infection

**Synonyms**
Congenital HPV infection 

**Definition (free text, please give PubMed ID)**
Human papillomavirus infection in neonates due to maternal transmission.
DOI: https://doi.org/10.1155/2010/326369

**Parent term (use hpo.jax.org/app)**
Persistent human papillomavirus infection

**Comment by pnrobinson** [comment]
This is similar to the this term, I think it belongs in the PMH section. 
https://hpo.jax.org/app/browse/term/HP:0033624
added as HP:4000011
